docs/testing/overview.md

```diff
@@ -1,6 +1,52 @@
 # Testing Overview
 
-At MetaMask, we are committed to delivering high-quality software. A key aspect of achieving this goal is through a robust testing strategy that includes both end-to-end (e2e), integration and unit tests. This document provides comprehensive guidelines on each testing type, emphasizing the importance of choosing the right test for the right scenario and detailing the specific practices for writing and organizing these tests.
+At MetaMask, we are committed to delivering high-quality software. These guidelines describe how we use automated tests to maximize product quality while minimizing the cost of writing and maintaining tests.
+
+Our automated tests fall into three general categories: end-to-end (e2e), integration and unit tests. This document provides comprehensive guidelines on each testing type, emphasizing the importance of choosing the right test for the right scenario and detailing the specific practices for writing and organizing these tests.
+
+## Testing principles
```

Comment:
Maybe I should move this to a separate document, and link it here?

Reply:
I actually like this being inlined... A testing overview doc should be the entry/starting point for devs when they are writing tests for the first time in our codebase. So it is important to highlight what are our transversal testing principles before diving into the different types of tests.